api/mesh/v1alpha1/mux.proto

```diff
@@ -14,5 +15,7 @@ message Message {
   oneof value {
     envoy.api.v2.DiscoveryRequest request = 1;
     envoy.api.v2.DiscoveryResponse response = 2;
+    envoy.service.discovery.v3.DiscoveryRequest requestV3 = 3;
+    envoy.service.discovery.v3.DiscoveryResponse responseV3 = 4;
```

Comment:
I don't think the proto fields are in the message at all would it make sense to rename the field properly? Have the v3 be `request/response` and the v2 by `deprecated_request` or `legacy_` ?

Or simply put v2 too?

Reply:
Good point! I forgot that we don't convert this to JSON/YAML. Changed the old field to legacyRequest